src/platform/linux/x11/xdisplay.rs

```diff
@@ -45,12 +45,12 @@ impl XConnection {
         };
 
         Ok(XConnection {
-            xlib: xlib,
-            xf86vmode: xf86vmode,
-            xcursor: xcursor,
-            xinput2: xinput2,
-            xlib_xcb: xlib_xcb,
-            display: display,
+            xlib,
```

Comment:
Let's keep compatibility with older Rust versions if we can. Please don't use that new syntax.